docs/built-in-bpa-rules.md

```diff
@@ -22,4 +22,19 @@ Service Fabric clusters should only use Azure Active Directory for client authen
 ## Use built-in roles instead of custom RBAC roles
 You should only use built-in roles instead of cutom RBAC roles. Custom RBAC roles are error prone. Using custom roles is treated as an exception and requires a rigorous review and threat modeling.
 
-**Recommendation**: [Use built-in roles such as 'Owner, Contributer, Reader' instead of custom RBAC roles](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles)
\ No newline at end of file
+**Recommendation**: [Use built-in roles such as 'Owner, Contributer, Reader' instead of custom RBAC roles](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles)
+
+## API App Should Only Be Accessible Over HTTPS
+API Apps should require HTTPS to ensure connections are made to the expected server and data in transit is protected from network layer eavesdropping attacks.
+
+**Recommendation**: In the [Microsoft.Web/Sites resource properties](https://docs.microsoft.com/en-us/azure/templates/microsoft.web/sites?tabs=json#siteproperties-object), add (or update) the *httpsOnly* property, setting its value to `true`.
```

Comment:
We've been adding documentation where it talks about the recommendation in more detail too. Here is a link that is relevant for these https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-bindings#enforce-https

Reply:
I recommend something along the lines of

To [use HTTPS to ensure server/service authentication and protect data in transit from network layer eavesdropping attacks](https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-bindings#enforce-https), in the [Microsoft.Web/Sites resource properties](https://docs.microsoft.com/en-us/azure/templates/microsoft.web/sites?tabs=json#siteproperties-object), add (or update) the *httpsOnly* property, setting its value to `true`.

This way there's context on what you're doing and how to do it.